src/containerapp/azext_containerapp/custom.py

```diff
@@ -5218,3 +5206,220 @@ def create_connected_environment(cmd,
     r = connected_env_create_decorator.create()
 
     return r
+
+
+def connected_env_list_dapr_components(cmd, resource_group_name, environment_name):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
+
+    return ConnectedEnvDaprComponentClient.list(cmd, resource_group_name, environment_name)
+
+
+def connected_env_show_dapr_component(cmd, resource_group_name, dapr_component_name, environment_name):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
+
+    return ConnectedEnvDaprComponentClient.show(cmd, resource_group_name, environment_name, name=dapr_component_name)
+
+
+def connected_env_remove_dapr_component(cmd, resource_group_name, dapr_component_name, environment_name):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
+
+    try:
+        ConnectedEnvDaprComponentClient.show(cmd, resource_group_name, environment_name, name=dapr_component_name)
+    except Exception as e:
+        raise ResourceNotFoundError("Dapr component not found.") from e
```

Comment:
when no resource is found, we do not need to delete it, right? And I think RP already handles the cases